validate.go

```diff
@@ -128,6 +134,24 @@ func checkPlatform(platform rspec.Platform) {
 	logrus.Fatalf("Operation system '%s' of the bundle is not supported yet.", platform.OS)
 }
 
+func checkHooks(hooks rspec.Hooks) {
+	checkEventHookPaths("pre-start", hooks.Prestart)
+	checkEventHookPaths("post-start", hooks.Poststart)
+	checkEventHookPaths("post-stop", hooks.Poststop)
+}
+
+func checkEventHookPaths(hookType string, hooks []rspec.Hook) {
+	for _, hook := range hooks {
+		fi, err := os.Stat(hook.Path)
+		if err != nil {
+			logrus.Fatalf("Cannot find %s hook: %v", hookType, hook.Path)
+		}
+		if fi.Mode()&0111 != 0 {
+			logrus.Fatalf("The %s hook %v: is not executable", hookType, hook.Path)
```

Comment:
I think the comparison should be `== 0` (i.e. “if none of the executable bits are set”).